ci/banned.h.sh

```diff
@@ -12,7 +12,7 @@
 
 set -eu
 
-files=$(find src -iname '*.[ch]' -not -path "src/bin/lib/*")
+files=$(find src -iname '*.[ch]' -type f | git check-attr --stdin citus-style | grep -v ': unset$' | sed 's/: citus-style: set$//')
```

Comment:
Each time I see `grep -v | another-filter` I cringe internally and want to use `awk` instead. It would look like the following if we did use awk. What do you think?

``` shell
find src -iname '*.[ch]' -type f | git check-attr --stdin citus-style | awk -F ':' '! /: unset$/ {print $1}'
```